tests/llmcompressor/transformers/obcq/test_consecutive_runs.py

```diff
@@ -36,19 +41,29 @@ def _test_consecutive_runs(
             oneshot_device=self.device,
             clear_sparse_session=False,
         )
-        first_tiny_model = get_session_model()
+
+        first_model = AutoModelForCausalLM.from_pretrained(
```

Comment:
I think the purpose of this test to test consecutive runs *without* model reloading.

Reply:
The test currently loads the first oneshot model from disk when running the second oneshot run so the focus of this particular case is consecutive one shot runs after first loading from disk

Reply:
We can do without reloading, but the original test on sparseml (if i remember correctly) was to load a saved model. As Dipika said above, we couldnt do that, bc HFQuantizer takes care of loading logic. 

The use case was that we optimize, save. Then we load from disk and optimize.

Reply:
I misread the code